<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bitcoin Inspector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="MemPool_GraphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>991</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QLabel" name="MemPool_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MemPool</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Info_PlainText">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>20</y>
      <width>441</width>
      <height>871</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Info_label">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Blockchain_GraphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>991</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Blockchain_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Blockchain</string>
    </property>
   </widget>
   <widget class="QLabel" name="MemPoolCounter_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="MenuVerticalLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>20</width>
      <height>900</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlockchainCounter_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>2</width>
     <height>2</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
